src/c/uCLib_base

```diff
@@ -0,0 +1,127 @@
+/*
+ * RISC OS uCLib (Micro C Library)
+ * 
+ * purpouse: This is a small (micro) re-implementation of a generic
```

Comment:
purpose, not purpouse (here and elsewhere)

Reply:
Sorry, as usual typing docs and license documents on the fly while doing regular life things! So thanks a lot for always reviewing these man! :)